<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ZonesExtractor</class>
 <widget class="QDialog" name="ZonesExtractor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1002</width>
    <height>654</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QGraphicsView" name="graphicsView"/>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="25" column="0" colspan="2">
      <widget class="QLabel" name="label_12">
       <property name="text">
        <string>Power watershed</string>
       </property>
      </widget>
     </item>
     <item row="28" column="0" colspan="2">
      <widget class="Line" name="line_6">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="30" column="0" colspan="2">
      <widget class="QPushButton" name="cannyFloodPreviewButton">
       <property name="text">
        <string>Preview</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0" colspan="2">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Mean treshold</string>
       </property>
      </widget>
     </item>
     <item row="10" column="0" colspan="2">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Standard deviation treshold</string>
       </property>
      </widget>
     </item>
     <item row="12" column="0" colspan="2">
      <widget class="QPushButton" name="texturePreviewButton">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Preview</string>
       </property>
      </widget>
     </item>
     <item row="9" column="1">
      <widget class="QDoubleSpinBox" name="meanRGBSpinBox">
       <property name="value">
        <double>4.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QSpinBox" name="squareSizeBox">
       <property name="maximum">
        <number>10000</number>
       </property>
       <property name="value">
        <number>5</number>
       </property>
      </widget>
     </item>
     <item row="8" column="0" colspan="2">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Mean RGB treshold</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0" colspan="2">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Square size</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QDoubleSpinBox" name="meanTresSpinBox">
       <property name="value">
        <double>4.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="11" column="1">
      <widget class="QDoubleSpinBox" name="stdTresSpinBox">
       <property name="value">
        <double>4.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="3" column="0" colspan="2">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Texture search</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="35" column="0" colspan="2">
      <widget class="QPushButton" name="resetButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Reset</string>
       </property>
      </widget>
     </item>
     <item row="15" column="0">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Lower treshold</string>
       </property>
      </widget>
     </item>
     <item row="13" column="0" colspan="2">
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="19" column="0" colspan="2">
      <widget class="QPushButton" name="floodPreviewButton">
       <property name="text">
        <string>Preview</string>
       </property>
      </widget>
     </item>
     <item row="21" column="0" colspan="2">
      <widget class="Line" name="line_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="18" column="1">
      <widget class="QSpinBox" name="fupspinBox">
       <property name="value">
        <number>20</number>
       </property>
      </widget>
     </item>
     <item row="16" column="1">
      <widget class="QSpinBox" name="flospinBox">
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="value">
        <number>20</number>
       </property>
      </widget>
     </item>
     <item row="17" column="0" colspan="2">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>Upper treshold</string>
       </property>
      </widget>
     </item>
     <item row="36" column="0" colspan="2">
      <widget class="QPushButton" name="applyButton">
       <property name="text">
        <string>Apply</string>
       </property>
      </widget>
     </item>
     <item row="29" column="0" colspan="2">
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>Canny flooding</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="0" colspan="2">
      <widget class="Line" name="line_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="14" column="0" colspan="2">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Floodfill</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="prespectiveAndCuttingButton">
       <property name="text">
        <string>Cut</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>Do not cut</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0" colspan="2">
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>Perspective and cutting</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="32" column="0" colspan="2">
      <widget class="QPushButton" name="fillHolesButton">
       <property name="text">
        <string>Fill holes</string>
       </property>
      </widget>
     </item>
     <item row="31" column="0" colspan="2">
      <widget class="Line" name="line_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="34" column="0" colspan="2">
      <widget class="Line" name="line_5">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="24" column="0" colspan="2">
      <widget class="QPushButton" name="powerWatershedMaskpushButton">
       <property name="text">
        <string>Preview</string>
       </property>
      </widget>
     </item>
     <item row="22" column="0" colspan="2">
      <widget class="QLabel" name="label_11">
       <property name="text">
        <string>Power watershed mask</string>
       </property>
      </widget>
     </item>
     <item row="26" column="0" colspan="2">
      <widget class="QPushButton" name="powerWatershedpushButton">
       <property name="text">
        <string>Preview</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
